src/common/configuration/global-variable-configuration.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
It looks like this file is intended to be a rename of `window-variable-configuration.ts`, but it looks like the PR doesn't actually remove the old file (or its test file); is that intentional? It doesn't look like the old file is still used anywhere after this PR.

Reply:
good catch! I'll remove those.